code/modules/surgery/organs/stomach.dm

```diff
@@ -251,8 +251,9 @@
 		var/message = input(owner, "What message would you like to imprint on the deposit?", "Deposit message:") as text|null
 		if(!message)
 			return
-		log_game("[key_name(owner)] as [owner] made a new spore deposit with the message [message] at [world.time]. [key_name(owner)] located at [AREACOORD(owner)]")
-		message_admins("[key_name(owner)] as [owner] made a new spore deposit with the message [message] at [world.time]. [key_name(owner)] located at [AREACOORD(owner)]")
+		log_game("[key_name(owner)] as [owner] made a new spore deposit with the message '[message]' at [world.time]. [key_name(owner)] located at [AREACOORD(owner)]")
+		message_admins("[key_name(owner)] as [owner] made a new spore deposit with the message '[message]' at [world.time]. [key_name(owner)] located at [AREACOORD(owner)]")
```

Comment:
use `key_name_admin` for `message_admins`. do NOT give dchat access to ckeys. remove "as [owner]", that is redundant because key_name already includes their name. Do not include "at [world.time]" as log_game already includes this in its key. do not add a redundant key_name just to say their location, log_game already includes their location. it's fine in message_admins though.

Personally I found the admin messages really annoying and spammy, I would prefer no admin messages.

change log_game to `owner.log_message("", type=LOG_GAME)` to get the epic auto-key stuff